components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -374,7 +393,7 @@ public void deleteUser(String userId) throws NotFoundException, CharonException
      * @param limit  Counting value
      * @return true if pagination is possible, false otherwise
      */
-    private boolean canPaginate(int offset, int limit) {
+    private boolean canPaginate(int offset, Integer limit) {
```

Comment:
Is it necessary to change this signature

Reply:
fixed:8f8c20292c4fac76778a8aa95490675023d7d063